target/linux/ath79/dts/ar9341_tplink_tl-mr3420-v2.dts

```diff
@@ -0,0 +1,190 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9341.dtsi"
+
+/ {
+	model = "TP-Link TL-MR3420 v2";
+	compatible = "tplink,tl-mr3420-v2", "qca,ar9341";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &system;
+		led-failsafe = &system;
+		led-running = &system;
+		led-upgrade = &system;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&jtag_disable_pins>;
+
+		reset {
+			label = "Reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 17 GPIO_ACTIVE_HIGH>;
+			debounce-interval = <60>;
+		};
+
+		rfkill {
```

Comment:
In ar71xx this was described as WPS button:
https://github.com/openwrt/openwrt/blob/master/target/linux/ar71xx/files/arch/mips/ath79/mach-tl-wr841n-v8.c#L130-L135
Do we care about that?

Reply:
> > I tested using their GPIO settings on MR3420v2, it keeps getting stuck in openwrt emergency boot mode.
> 
> This is the symptom of incorrect gpio active_low value. If using gpio-keys-polled caused your router stuck in emergency mode, you should change reset button to active_low instead.

Same thing happened when I set it to active_low as well. 
Still stucked in emergency boot mode.
I used the original reference wr841n-v8/wr842n-v2 initially but it doesn't work at all